### PR TITLE
fix(ld-input): prop forwarding

### PR DESCRIPTION
**Pull request by renet**
References:
  - #157

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] unit tests
- [x] tested manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/utils/__mocks__/cloneAttributes.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { JSXBase } from '@stencil/core/internal'
+import { MockAttr, MockHTMLElement } from '@stencil/core/mock-doc'
+
+export const cloneAttributes = <B extends MockHTMLElement>(
+  el: B,
+  attributesToIgnore: string[] = []
+) =>
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```
Comment (borisdiakur) [inline src/liquid/utils/__mocks__/cloneAttributes.ts:9]: What exactly are we ignoring here?
> It is recommended practice to have the remainder of the comment following @ts-ignore explain which error is being suppressed.

**Review by borisdiakur — CHANGES_REQUESTED** [review]

File: src/liquid/utils/__mocks__/cloneAttributes.ts (hunk 1 above)
Comment (renet) [reply src/liquid/utils/__mocks__/cloneAttributes.ts:10]: Done.

**Pull request merged**